utils/messageqcpp/messagequeuepool.cpp

```diff
@@ -25,40 +25,57 @@
 #include <new>
 #include <type_traits>
 
-
 namespace messageqcpp
 {
 
 using ClientMapType = std::multimap<std::string, std::unique_ptr<ClientObject>>;
 
-struct LockedClientMap
+template <class T>
```

Comment:
This nifty to ston change might affect performance in an unexpected manner. Need to pay attention.